client/packages/system/src/Patient/Components/ChipTableCell.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  Box,
+  CellProps,
+  RecordWithId,
+  Typography,
+} from '@openmsupply-client/common';
+import React from 'react';
+
+/**
+ * Displays a list of labels in a table cell.
+ *
+ * This expects that the data accessor returns a list of string for a given table row.
+ */
+
+export const ChipTableCell = <T extends RecordWithId>({
+  rowData,
+  column,
+}: CellProps<T>) => {
+  const items = column.accessor({ rowData }) as string[];
+
+  if (items.length === 0) return null;
+
+  return (
+    <Box
+      sx={{
+        flexDirection: 'row',
+        borderBottom: 'none',
+        alignItems: 'center',
+        display: 'flex',
+        flexWrap: 'wrap',
+      }}
+    >
+      {items.map((item, index) => (
+        <Box
+          sx={{
+            padding: 0.5,
+          }}
+          key={index}
+        >
+          <Typography
+            sx={{
+              fontSize: 12,
```

Comment:
nit: can the fontSize be smaller? If the patient is enrolled in a lot of programs the column length gets quite long... Though, happy to leave as is if 11 is too small
![Screenshot 2023-07-05 at 11 07 04 AM](https://github.com/openmsupply/open-msupply/assets/61820074/0220d2e9-37d3-43e2-9310-25e7b438ee6d)

Reply:
No actually had it at 11 but changed because you used 12 :) I just changed it to 11. I also use the Chip MUI component now:

![image](https://github.com/openmsupply/open-msupply/assets/88299195/7a7ba64e-1320-48d4-a548-24b1ee9d7461)

(Wanted to use it before but imported it wrong and then though we are using the wrong MUI version...)

Reply:
Much better!

Reply:
I played with the search ID label, but it makes it too long because it searches by NID, NUIC and enrolment ID and fitting all this in is too long...

Note, we need to think about a way how to make the ID labels configurable because currently its hardcoded for PNG...